Improve PEcAn.SIPNET documentation and connect to PEcAn book (#3705)
PecanProject/pecan: book_source/03_topical_pages/93_installation/03_install_OS/06_install_models/install_SIPNET.Rmd

#### SIPNET {#inst-sipnet}

> The recommended method to install SIPNET is from the official repository:

Reviewer: ```suggestion
You can download a compiled binary from the [SIPNET releases page](https://github.com/PecanProject/sipnet/releases). 

You can also install SIPNET is from the repository:

```

```bash
# Clone the SIPNET repository
git clone https://github.com/PecanProject/sipnet.git
cd sipnet

# Build SIPNET
make

# Install the executable (optional - adjust path as needed)
sudo cp sipnet /usr/local/bin/sipnet
```

For detailed installation instructions, build options, and dependencies, please see the [SIPNET Installation Guide](https://pecanproject.github.io/sipnet/install.html).

> **Note:** Previous installation methods using tarballs (`sipnet_unk.tar.gz`, `sipnet_r136.tar.gz`) are deprecated. Please use the repository method above for the most up-to-date version.

Reviewer: ```suggestion
**Note:** Previous installation methods using tarballs (`sipnet_unk.tar.gz`, `sipnet_r136.tar.gz`) are deprecated. Please use the repository method above for the most up-to-date version. All previous tagged releases are available as source code and compiled binaries from the SIPNET releases page.
```

#### SIPNET {#inst-sipnet}

You can download a compiled binary from the [SIPNET releases page](https://github.com/PecanProject/sipnet/releases). 

You can also install SIPNET is from the repository:

```bash
# Clone the SIPNET repository
git clone https://github.com/PecanProject/sipnet.git
cd sipnet

# Build SIPNET
make

# Install the executable (optional - adjust path as needed)
sudo cp sipnet /usr/local/bin/sipnet
```

For detailed installation instructions, build options, and dependencies,
please see the [SIPNET Installation Guide](https://pecanproject.github.io/sipnet/install.html).

**Note:** Previous installation methods using tarballs (`sipnet_unk.tar.gz`, `sipnet_r136.tar.gz`) are deprecated.
Please use the repository method above for the most up-to-date version.
All previous tagged releases are available as source code and compiled binaries from the SIPNET releases page.

> **SIPNET testrun**

Reviewer 2: ````suggestion
**SIPNET testrun**

```bash
/usr/local/bin/sipnet --version
/usr/local/bin/sipnet --help
```
````
